staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -532,7 +529,7 @@ func (cfgCtlr *configController) processNextWorkItem() bool {
 // objects that configure API Priority and Fairness and updates the
 // local configController accordingly.
 // Only invoke this in the one and only worker goroutine
-func (cfgCtlr *configController) syncOne() (specificDelay time.Duration, err error) {
+func (cfgCtlr *configController) syncOne(ctx context.Context) (specificDelay time.Duration, err error) {
 	klog.V(5).Infof("%s syncOne at %s", cfgCtlr.name, cfgCtlr.clock.Now().Format(timeFmt))
```

Comment:
Let's use the context's logger.

Reply:
Out of scope to change everything to contextual logging. Follow-up please.

Reply:
I do not understand why all the other switches to contextual logging are in scope.
I am fine with breaking this work up into multiple smaller PRs.

Reply:
Because I touched the logger code for some other reason. But yet, could have been kept out.